packages/myst-common/src/ruleids.ts

```diff
@@ -0,0 +1,95 @@
+export enum RuleId {
+  // Frontmatter rules
+  validConfigStructure = 'valid-config-structure',
+  siteConfigExists = 'site-config-exists',
+  projectConfigExists = 'project-config-exists',
```

Comment:
Very cool to see 80+ rules come into existence for this. Wow. :)

Reply:
Hah, yeah, I'm not quite sure what the correct granularity is for these guys, but for now I think this works!